### PR TITLE
Added README for scrubs

### PR DESCRIPTION
**Pull request by rjmcneill**

### CODE REVIEW

**Comment by rjmcneill** [comment]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+#### Build the backend:
+
+`./gradlew build`
+
+#### Run the backend:
+
+`java -jar backend/build/libs/tweetscuttler-0.1.1.jar`
+
+#### In a new terminal server the frontend:
```
Comment (jdhodges) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+`backend/src/main/resources/application.properties`
+
+#### Build the backend:
+
+`./gradlew build`
+
+#### Run the backend:
+
+`java -jar backend/build/libs/tweetscuttler-0.1.1.jar`
```
Comment (stevenjob) [inline README.md]: ### build and run the backend

./gradlew bootRun

does this still work?

Comment (jdhodges) [inline README.md]: is any of this real?

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+`backend/src/main/resources/application.properties`
+
+#### Build the backend:
+
+`./gradlew build`
+
+#### Run the backend:
+
+`./gradlew bootRun`
```
Comment (stevenjob) [inline README.md:14]: this also builds it i think

**Pull request merged**